x-pack/plugins/security_solution/public/detections/components/host_isolation/use_host_isolation_action.tsx

```diff
@@ -131,16 +140,17 @@ export const useHostIsolationAction = ({
 
   const isIsolationActionDisabled = useMemo(() => {
     if (sentinelOneManualHostActionsEnabled && isSentinelOneAlert) {
-      return (
-        !sentinelOneAgentStatus ||
-        sentinelOneAgentStatus?.isUninstalled ||
-        sentinelOneAgentStatus?.isPendingUninstall
-      );
+      // 8.14 use FF for computing if action is enabled
+      return !sentinelOneAgentStatus || agentStatusClientEnabled
+        ? sentinelOneAgentStatus?.status === HostStatus.UNENROLLED
+        : (sentinelOneAgentStatus as AgentStatusInfo[string])?.isUninstalled ||
+            (sentinelOneAgentStatus as AgentStatusInfo[string])?.isPendingUninstall;
```

Comment:
personally I find this section pretty hard to read. Extracting some of these things to variables would make it a bit easier. Small thing though, no big deal!

Reply:
d078d7dbb172690b746ac0ab225e733698669e60